server_addon/unreal/package.py

```diff
@@ -1,3 +1,3 @@
 name = "unreal"
 title = "Unreal"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Not needed when nothing changes on server side.
```suggestion
version = "0.1.0"
```